sql/mob_groups.sql

```diff
@@ -377,7 +377,7 @@ INSERT INTO `mob_groups` VALUES (330,1720,11,0,128,0,0,0,28,31,0);
 INSERT INTO `mob_groups` VALUES (331,1739,11,924,0,1490,0,0,42,43,0);
 INSERT INTO `mob_groups` VALUES (332,1746,11,0,128,1511,7400,0,50,55,0);
 INSERT INTO `mob_groups` VALUES (333,1816,11,0,128,0,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (334,2685,11,924,0,0,0,0,45,45,0);
+INSERT INTO `mob_groups` VALUES (334,2685,11,924,0,2315,0,0,45,45,0);
```

Comment:
Droplists containing Moblin Hotrok seem off in our DB.  I see two dropids that contain the Hotrok: 2314 and 2315.

2315 is assigned to Moblin_Gurneyman and Armoury_Crate in Oldton, and has a buncha other Moblin armor in the droplist.  Hotrok shouldn't drop from those.

2314 is just the Hotrok and a Moblin Helm, and it's assigned to Moblin_Groundman -- who are supposed to drop the Hotrok -- but 2314 is also assigned to Flame Giant in Newton for some reason.

To finish the weirdness, NM Roo_Beju_the_Eulogizer (not yet implemented) is currently assigned groupid 334, same as the Moblin Ashmen.  We should prolly set its groupid to 0 until it's implemented and gets one of its own.

None of the droplists seem to match the Groundman and Ashman drops completely.
http://ffxidb.com/items/1729

😕🔥💻

Reply:
I say blow'em all away and reuse the ID's while creating actual accurate drop lists for these mobs.
:boom: :gun:

Reply:
Certainly good food for thought.  I have not had to maintain a public repo.  Thanks.

Reply:
I opened a PR #5646 just now that should fix the shared groups (doesn't fix the droplists, tho)

Reply:
Drop lists should be fixed in this fork.  Should I update with the IDs from your PR?  Or should this PR depend on yours?

Reply:
mm if my PR gets merged before yours, I'd rebase onto the latest master at that point.  looks like you'd use my sql files wholesale, but then add on your new droplist (at first available dropID 4653 rather than 4646) and update the dropid to 4653 on that Ashman groupid (334)

Reply:
Got all the content from DSP master into my fork.  Looking good, no conflicts.